change_log.md

```diff
@@ -1,8 +1,15 @@
 # OWASP Java HTML Sanitizer Change Log
 
 Most recent at top.
-  * Pending
-    * Fix table formatting
+  * Release 20200713.1
+    * Do not lower-case SVG/MathML names.
+      This shouldn't cause problems since it was hard to write policies for
+      SBG, but be aware that SVG's `<textArea>` is now distinct from HTML's `<textarea>`.
+  * Release 20200615.1
+    * Change `.and` when combining two policies to respect explicit `skipIfEmpty` decisions.
+    * HTML entity decoding now follows HTML standard rules about when a semicolon is optional.
+      [Fixes #193](https://github.com/OWASP/java-html-sanitizer/issues/193)
+    * Fix table formatting [#137](https://github.com/OWASP/java-html-sanitizer/issues/137)
```

Comment:
Most of the edits seem to be merge edits which makes this hard to review.  Did you have a branch that was pointed at an older version?  Can you reapply your patch to a new branch derived from HEAD?